src/iadmin.cpp

```diff
@@ -2516,6 +2660,40 @@ usage( char *subOpt ) {
         ""
     };
 
+    char* set_pam_password_config_usage[] = {
+        "set_pam_password_config OPTION_NAME OPTION_VALUE",
+        " ",
+        ("Set the specified OPTION_NAME to OPTION_VALUE for pam_password configuration in the local zone in "
+         "R_GRID_CONFIGURATION."),
+        " ",
+        // TODO: Add descriptions for these option names...
+        "The list of accepted OPTION_NAMEs is:",
+        " - password_length",
+        " - no_extend",
+        " - password_min_time",
+        " - password_max_time",
+        ""};
+
+    char* get_pam_password_config_usage[] = {
+        "get_pam_password_config",
+        " ",
+        "Prints pam_password configuration from the local zone grid configuration as JSON.",
+        " ",
+        ("This command allows administrators to view pam_password configuration values without running queries "
+         "directly against the database."),
+        " ",
+        "This information is retrieved from the R_GRID_CONFIGURATION database table.",
+        " ",
+        R"_(Example Output:
+
+    {
+        "password_length": 5000,
+        "no_extend": 0,
+        "password_min_time": 3600,
+        "password_max_time": 360000
+    })_",
+        ""};
```

Comment:
clang-format betrayed me

Reply:
Lol, disable clang-format for that part.

Reply:
Actually I think this is right. It at least looks like the `get_delay_server_info` help text, so I'm just going to leave it.